src/components/Layout.tsx

```diff
@@ -24,6 +25,12 @@ export function Layout() {
     ...(matches.find((match) => Boolean(match.handle))?.handle || {})
   } as RouteHandleObject;
 
+  const md = useMediaQuery(theme.breakpoints.only('md'));
+  const lg = useMediaQuery(theme.breakpoints.only('lg'));
+  const xl = useMediaQuery(theme.breakpoints.only('xl'));
+
+  const sidePadding = sidebar.visible ? 3 : xl ? 34.6 : lg ? 27.3 : md ? 12 : 3;
```

Comment:
Here we don't need to use `useMediaQuery`, and we also should avoid the nested ternary operations for readability purpose.

```const sidePadding = sidebar.visible ? 3 : { xs: 3, md: 12, lg: 27, xl: 34 };```